example/README.md

```diff
@@ -0,0 +1,171 @@
+# Wattsight API python library usage example
+An introductory example of using Wattsight API python library for searching, fetching, and plotting Wattsight data series. A number of use cases is presented in order to introduce different data structures.
+
+## Dependencies
+
+[Wattsight API python library](https://github.com/wattsight/wapi-python/)   
+[Jupyter (iPython) Notebooks](http://jupyter.org/)   
+[Pandas Data Analysis library](http://pandas.pydata.org/)   
+[Plotly Charting library](https://plot.ly/)   
+[World Timezone Definitions for Python](http://pytz.sourceforge.net/)   
+
+## Installation of Wattsight API python library
+
+1. Using curl
+
+	```command
+	curl -LOk https://github.com/wattsight/wapi-python/archive/wapi-python-0.0.7.zip
+	unzip wapi-python-0.0.7.zip
+	cd wapi-python-wapi-python-0.0.7
+	python setup.py install
+	```
+
+2. Using pip
+
+	```command
+	pip install git+https://github.com/wattsight/wapi-python@wapi-python-0.0.7
+	```
+
+## Implementation
+Complete working code can be found in the attached Jupyter Notebook file.
+
+1. Established a session using client cridentials. 
+
+	This requires having an OAuth client, which can be created at https://auth.wattsight.com/account/oauth-clients.
+
+	```python
+	>>> session = wapi.Session(client_id='client id', client_secret='client secret')
+	```
+
+2. Search curves by meta data.
+
+	For this use case define meta criteria for DK1 area wind power production.
+
+	```python
+	>>> meta_criteria = {'commodity': 'POW',
+	                     'category': ['WND', 'Intraday'], 
+	                     'unit': 'MWH/H',
+	                     'area': 'DK1', 
+	                     'data_type': ['A', 'F']}
+	>>> curves = session.search(**meta_criteria)
+	['pro dk1 wnd intraday ec00da mwh/h cet min15 f',
+	 'pro dk1 wnd intraday lastec mwh/h cet min15 f',
+	 'pro dk1 wnd intraday tso mwh/h cet min15 f',
+	 'pro dk1 wnd intraday mwh/h cet min15 a',
+	 'pro dk1 wnd ec00 mwh/h cet min15 f',
+	 'pro dk1 wnd ec12 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs00 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs06 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs12 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs18 mwh/h cet min15 f',
+	 'pro dk1 wnd ec00ens mwh/h cet min15 f',
+	 'pro dk1 wnd ec12ens mwh/h cet min15 f',
+	 'pro dk1 wnd mwh/h cet min15 a']
+	```
+
+3. For the first example, filter on intraday curves and retrieve data for the selected curves.
+
+	```python
+	>>> intraday_curves = [c for c in curves if 'INTRADAY' in c.categories]
+	>>> data = {}
+	>>> label = {}
+	>>> issue_date = pytz.timezone('CET').localize(
+						datetime.strptime('2018-04-12 00:00', "%Y-%m-%d %H:%M"))
```

Comment:
This is again quite complex, I'm certain it can be done in a far more readable way.  Esp. as you hardcode the date, why not just create one with year/month/day params directly?  strptime should not be needed.